R/converters_SkylinetoMSstatsFormat.R

```diff
@@ -66,7 +66,7 @@ SkylinetoMSstatsFormat = function(
                        score_threshold = qvalue_cutoff, 
                        direction = "smaller",
                        behavior = "fill", 
-                       fill_value = 0, 
+                       fill_value = NA_real_, 
```

Comment:
Why is the fill_value `NA_real_` here but `NA` for Spectronaut?

Reply:
idk! Just following the behavior from further up in the Skyline code.

Apparently:

"In R, NA_real_ is a typed missing value that represents a double, or value with a decimal point, that is missing. NA is the general symbol used to represent missing values in R, and R automatically converts NA to the appropriate typed missing value for the target vector. "

I'm guessing this really doesn't have an impact? Maybe we just go with NA? 

@mstaniak do you have a reason to use NA_real_?

Reply:
Some packages (for example dplyr-related) check data types very strictly, including NA types. It should be NA_real_ everywhere